wgpu/src/lib.rs

```diff
@@ -3151,13 +3161,14 @@ impl<'a> ComputePass<'a> {
 
 /// [`Features::PUSH_CONSTANTS`] must be enabled on the device in order to call these functions.
 impl<'a> ComputePass<'a> {
-    /// Set push constant data.
+    /// Set push constant data for subsequent dispatch calls.
```

Comment:
These docs should reference the other docs

Reply:
Discussed in chat - the constraints on push constants in render passes don't apply to compute passes, so these docs are okay.